.travis.yml

```diff
@@ -17,7 +17,7 @@ env:
         # to repeat them for all configurations.
         - PYTHON_VERSION=3.6
         - CONDA_DEPENDENCIES='cython astropy sphinx'
```

Comment:
the default render of the template has astropy as a install_requires and C code. So it needs it for the test build.